esp-metadata-generated/src/_generated_esp32.rs

```diff
@@ -701,6 +701,9 @@ macro_rules! define_clock_tree_types {
             rc_fast_clk_refcount: u32,
             apb_clk_refcount: u32,
             ref_tick_refcount: u32,
+            xtal32k_clk_refcount: u32,
+            rc_slow_clk_refcount: u32,
+            rc_fast_div_clk_refcount: u32,
```

Comment:
These are all possible TIMGn calibration clock sources, which means the system no longer turns these clocks off after calibration if they are otherwise still in use.